### DISCUSSION TITLE
Create a separate build that signs the dll with a strong name key

### DISCUSSION THREAD

**Discussion by abc983ft** [desjarlais/Scintilla.NET#100] [Ideas]
You previously had a build that signed with a strong name key: https://www.nuget.org/packages/unofficial.ScintillaNET.Signed

It would be great if you create a signed build for the current version

**Comment by VPKSoft** [comment]
Hi,
Where would this be useful? This would lead to an issue where the satellite packages would also have to have two versions and so on. It is also specifically not recommended to add this on an existing package: https://learn.microsoft.com/en-us/dotnet/standard/library-guidance/strong-naming

**Comment by abc983ft** [comment]
It is needed for COM interop.
See https://github.com/jacobslusser/ScintillaNET/issues/11#issuecomment-103313141

**Comment by desjarlais** [comment]
I will look at a separate build for signed scenarios like COM Interop, but that might take some time to get done.

**Comment by desjarlais** [comment]
@abc983ft out of curiosity, is there any particular reason you don't want to use the workaround suggested by jacob?  

"While that's being debated I might suggest you fork the ScintillaNET repo and add your strong-name support for anyone who wants it"

**Comment by desjarlais** [comment]
This has been added to version 5.6.2.
